src/qt/forms/sendcoinsdialog.ui

```diff
@@ -1108,7 +1108,7 @@
            <item>
             <widget class="QCheckBox" name="optInRBF">
              <property name="text">
-              <string>Request Replace-By-Fee</string>
+              <string>Allow increasing transaction fee later on</string>
```

Comment:
Even though within the gui you can only do fee bumping as a result of RBF, in reality RBF allows for more than just increasing the fee. (e.g. additional recipients)

So I think you should at least mention BIP-125 replaceable in parenthesis.

Reply:
Maybe remove "on"?